tool/common/update/feature_ent.go

```diff
@@ -0,0 +1,26 @@
+//go:build webassets_ent
```

Comment:
Would be better to introduce new build tag for enterprise version instead of reusing this one for web assets?